src/scss/components/_hero.scss

```diff
@@ -0,0 +1,125 @@
+.hero {
+    background-color: var(--bisque);
+    display: flow-root;
+    color: var(--black);
+
+    &__wrapper{
+        position: relative;
+        height: fit-content;
+        width: 100%;
+    }
+
+    &__title{
```

Comment:
Add `text-transform: uppercase;` so that the text is always uppercased

Reply:
Fixed